src/aleph_client/asynchronous.py

```diff
@@ -301,25 +347,52 @@ async def create_program(
         entrypoint: str,
         runtime: str,
         environment_variables: Optional[Dict[str, str]] = None,
-        storage_engine: StorageEnum = StorageEnum.storage,
-        channel: str = settings.DEFAULT_CHANNEL,
-        address: Optional[str] = settings.ADDRESS_TO_USE,
+        storage_engine: Optional[StorageEnum] = None,
+        channel: Optional[str] = None,
+        address: Optional[str] = None,
         session: Optional[ClientSession] = None,
-        api_server: str = settings.API_HOST,
-        memory: int = settings.DEFAULT_VM_MEMORY,
-        vcpus: int = settings.DEFAULT_VM_VCPUS,
-        timeout_seconds: float = settings.DEFAULT_VM_TIMEOUT,
-        persistent: bool = False,
-        encoding: Encoding = Encoding.zip,
+        api_server: Optional[str] = None,
+        memory: Optional[int] = None,
+        vcpus: Optional[int] = None,
+        timeout_seconds: Optional[float] = None,
+        persistent: Optional[bool] = None,
```

Comment:
Revert.

Reply:
What exactly?

Reply:
Same discussion as above, but `persistent: bool = False` is fine.